src/sentry/api/endpoints/organization_member/__init__.py

```diff
@@ -83,11 +83,12 @@ def can_set_team_role(request: Request, team: Team, new_role: TeamRole) -> bool:
 
 
 def can_admin_team(access: Access, team: Team) -> bool:
-    if access.has_scope("org:write"):
-        return True
-    if not access.has_team_membership(team):
-        return False
-    return access.has_team_scope(team, "team:write") or access.has_team_scope(team, "team:admin")
+
+    return access.has_team_membership(team) and (
+        access.has_team_scope(team, "team:write")
+        or access.has_scope("org:write")
+        or access.has_scope("member:write")
```

Comment:
main change
1. If they dont have team membership -> deny
    - team membership is determined via -> the token's attached organization is the same as the requested team's organization and the team is active ([CODE](https://github.com/getsentry/sentry/blob/crl/support-editing-team-token/src/sentry/auth/access.py#L706-L708))
and one of
1. token has org: write
3. token has member:write
4. has team:write scope and within the team
   - determined via -> do the same team membership check as above, does the token have the given scope ([CODE](https://github.com/getsentry/sentry/blob/master/src/sentry/auth/access.py#L503-L507))